scripts/compare_remote.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import requests
+import json
+import argparse
+
+parser = argparse.ArgumentParser(description='Compare current against Shavar or Disconnect.me blocklist. Default compare against remote.')
+parser.add_argument('-d', '--disconnect', help='Compare against Disconnect', action='store_true')
+parser.add_argument('-s', '--shavar', help='Compare against Disconnect')
+args = parser.parse_args()
+if args.disconnect:
+    github_url = 'https://raw.githubusercontent.com/disconnectme/shavar-prod-lists/master/disconnect-blacklist.json'
+else:
```

Comment:
You should check explicitly if this is shavar since you add it as an argument, and can throw an error otherwise.

Reply:
Hmm, I wanted to compare the Shavar remote as a default... Should I update this to remove the default and explicitly state Shavar in the command's argument?